### DISCUSSION TITLE
Aprender Vuejs ou Reactjs tendo como objetivo salario

### DISCUSSION THREAD

**Discussion by francysreymer** [frontendbr/forum#1864] [Discussão]
Ola pessoal,

gostaria de saber da opinião do pessoal aqui no grupo sobre qual e melhor tendo em vista unicamente o salario, Vuejs ou Reactjs?

Ja estou no mercado ha um bom tempo no backend com PHP e estou atualmente iniciando um projeto na empresa com Vuejs, Firebase, Nodejs e Typescript mas seguindo algumas vagas em redes sociais e aqui mesmo vejo bem mais Reactjs. Qual a opiniao do pessoal que trabalha aqui com o frontend sobre melhor remuneracao?

**Comment by helderberto** [comment]
> Ola pessoal,
> 
> gostaria de saber da opinião do pessoal aqui no grupo sobre qual e melhor tendo em vista unicamente o salario, Vuejs ou Reactjs?
> 
> Ja estou no mercado ha um bom tempo no backend com PHP e estou atualmente iniciando um projeto na empresa com Vuejs, Firebase, Nodejs e Typescript mas seguindo algumas vagas em redes sociais e aqui mesmo vejo bem mais Reactjs. Qual a opiniao do pessoal que trabalha aqui com o frontend sobre melhor remuneracao?

Olá!

Alguns pontos que você pode levar em consideração são:
- Quais empresas você almeja fazer parte e quais tecnologias a empresa esta utilizando?
- Pesquisas globais como [Stack Overflow 2020](https://insights.stackoverflow.com/survey/2020#technology-most-loved-dreaded-and-wanted-web-frameworks) e quais bibliotecas/frameworks estão sendo mais utilizados

Pessoalmente, não acredito que exista uma "melhor" opção e sim maior demanda de mercado e o que você busca.

Espero ter ajudado. :D

**Comment by ninetails** [comment]
Adicionando, também tem a [State of JS do ano passado](https://2019.stateofjs.com/front-end-frameworks/).

Concordo com o @helderburato , em questão de empregabilidade porém vejo no Brasil muito mais vagas pedindo React e Angular que Vue.js. Mas o que eu recomendaria é se vc não tiver um framework de preferência, analisar então os lugares onde você quer trabalhar, fazer um prospecto de tais vagas e decidir daí estudar framework voltado às vagas que mais te chamarem atenção (e levando em conta a remuneração destas vagas).

**Comment by nandomoreirame** [comment]
Aprenda **JavaScript**, os frameworks são apenas ferramentas e o salário vai depender da empresa.

Abraços.

**Comment by gabriel-hahn** [comment]
Complementando os comentários anteriores, acredito que tu pode ter um ótimo salário independente de qual framework/biblioteca tu escolher aprender. Realmente React possui mais vagas em aberto (pelo menos aqui no Brasil e na Europa, baseado em vagas no Stackoverflow, Linkeding e etc) e tua 'chance' de conseguir uma vaga boa com React parece ser maior, porém salário é bem relativo dependendo da região que tu mora, empresa e principalmente teu grau de conhecimento nas tecnologias, ou seja, desenvolve algo próprio utilizando esses caras e vê com qual tu se identifica mais e curte trabalhar. Independente da tecnologia, com o tempo e teu gosto por trabalhar com ela, o salário e boa remuneração vem como consequência do teu empenho.
Acho que tecnologia versus salário é algo que não é tão certo comparar hoje em dia, pois existe muita gente trabalhando com Cobol e ganhando uma grana por ai, mas nem por isso eu diria que é tããão indicado aprender essa tecnologia haha
Espero ter ajudado, mesmo não sendo uma resposta objetiva haha abraço

**Comment by nathpaiva** [comment]
Acredito que melhor do que escolher algo pensando no dinheiro, é escolher algo que você realmente curta.

Porque oportunidade com qualquer uma das duas, você terá. Existem boas vagas tanto para Vue quanto para React, dentro ou fora do Brasil.

Então minha dica, escolha algo que você se sinta a vontade em aprender, e se dedique, se envolva com a comunidade, faça contatos e amigos, e claro, com uma boa base do JavaScript, fica fácil mudar de uma tecnologia para a outra. 😉
